src/stretchy.js

```diff
@@ -223,17 +235,16 @@ const svgSpan = function(group, options) {
             svgNode = new domTree.svgNode([path], attributes);
 
             if (numSvgChildren === 1) {
-                span = buildCommon.makeSpan(["hide-tail"], [svgNode], options);
+                spans.push(buildCommon.makeSpan(["hide-tail"], [svgNode], options));
```

Comment:
What if we just return `buildCommon.makeSpan(["hide-tail"], [svgNode], options)` here immediately?  Then we wouldn't have to special case things below.

Reply:
We can't return the span yet at that point. We still have to set some height and min-width properties later in the function.

Reply:
My bad, I didn't see that github had collapsed some of the code.  Nevermind.